scripts/sanitycheck

```diff
@@ -573,6 +573,10 @@ structure in the main Zephyr tree: boards/<arch>/<board_name>/""")
         "--device-serial",
         help="Serial device for accessing the board (e.g., /dev/ttyACM0)")
 
+    parser.add_argument(
+        "--device-serial-pty",
+        help="Serial pty script for creating pseudoterminal")
```

Comment:
Is there documentation somewhere? What is this script supposed to do? Could you include a sample?

Reply:
Basically the stdout of the script would be pseudoterminal input.